spec/views/devise/registrations/edit.html.slim_spec.rb

```diff
@@ -1,37 +0,0 @@
-require 'rails_helper'
-
-describe 'devise/registrations/edit.html.slim' do
-  before do
-    @update_user_profile_form = UpdateUserProfileForm.new(build_stubbed(:user))
-  end
-
-  it 'has a localized title' do
-    expect(view).to receive(:title).with(t('upaya.titles.registrations.edit'))
-
-    render
-  end
-
-  it 'sets form autocomplete to off' do
-    render
-
-    expect(rendered).to have_xpath("//form[@autocomplete='off']")
-  end
-
-  it 'contains link to enable TOTP' do
-    render
-
-    expect(rendered).to have_xpath("//input[@value='Enable']")
-    expect(rendered).not_to have_xpath("//input[@value='Disable']")
-  end
-
-  context 'when user is TOTP enabled' do
-    it 'contains link to disable TOTP' do
-      user = build_stubbed(:user, otp_secret_key: '123')
-      @update_user_profile_form = UpdateUserProfileForm.new(user)
-      render
-
-      expect(rendered).to have_xpath("//input[@value='Disable']")
-      expect(rendered).not_to have_xpath("//input[@value='Enable']")
-    end
-  end
-end
```

Comment:
I had to move the views to the proper location to allow the tests to pass. They were failing because they were looking for a `devise/registrations` controller, but it's under `users/registrations` in our app.